src/layerInfos/NetzentwicklungLayerInfo/NetzentwicklungLayerInfo.js

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+
+const cartaroURL = process.env.REACT_APP_CARTARO_URL;
+
+const useStyles = makeStyles({
+  title: {
+    margin: '5px 0',
+  },
+  regionLegendItem: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  regionColor: {
+    width: 30,
+    height: 20,
+    margin: 5,
+    marginLeft: 0,
+    border: '1px solid #767676',
+  },
+});
+
+const propTypes = {
+  layerName: PropTypes.string.isRequired,
+  language: PropTypes.string.isRequired,
+  t: PropTypes.func.isRequired,
+};
+
+const topicInfo = {
```

Comment:
@gevos Since we made this topic public (and don't ingest it into trafimage from outside) we could move all the translations to the translation files and do the language handling in i18n. That would make the topic more consistent with the other public topics.

Reply:
Done